### ISSUE TITLE
Double tap Apple pencil to switch between drawing and erasing

### ISSUE THREAD

**Issue by Jeyenkc** [zsviczian/obsidian-excalidraw-plugin#562]
The Apple pencil comes has a built in shortcut into it that allows it to perform a function. It is almost always bound to toggling between drawing/writing and erasing. This plugin currently does not support this feature, but it's a huge QoL feature that I cannot do without anymore.

Using an Apple tablet and Apple pencil, I want to be able to switch between drawing and erasing when I double tap the pencil. On apps like Notability, double tapping the pencil toggles between the eraser and the currently selected tool.

It would be great to have this feature! Love this plugin. Great work!

**Comment by zsviczian** [comment]
I agree this would be very nice... but it is unlikely to happen. Unfortunately, there are many different styluses and there isn't a standard for how their buttons are registered. I've looked into this a bit a few weeks ago, but there wasn't an obvious solution available for recognizing the mouse tap in a web app (which obsidian and excalidraw technically is). I am sorry. 

I will keep searching for a viable solution, but with limited hope.

**Comment by XDanielPaul** [comment]
I know it still might not be viable, but wouldn't it be possible to look for an option after 4 years?

**Comment by zsviczian** [comment]
Updated the umbrella FR: #2773 to include this one as well.
As said in #2773, I welcome contributions in the form of PRs. That said, I doubt this can be solved, since to the best of my understanding pencil support requires SDK which would need to be built into the Obsidian app first.
In plugin settings you can set up double tap (finger on the screen) to change between eraser and tool. This works for all styluses, not limited to Apple.
